docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -393,6 +394,63 @@ inputs = {
 }
 ```
 
+**What is deep merge?**
+
+When the `merge_strategy` for the `include` block is set to `deep`, Terragrunt will perform a deep merge of the included
+config. For Terragrunt config, deep merge is defined as follows:
+
+- For simple types, the source overrides the target.
```

Comment:
NIT: What is source and what is target? Would help to clarify, perhaps with the example mentioned above.

Reply:
Switched to child and parent language: https://github.com/gruntwork-io/terragrunt/pull/1764